xclim/indices/_threshold.py

```diff
@@ -2111,25 +2172,21 @@ def snd_season_length(
 
     Number of days where surface snow depth is greater or equal to given threshold (default: 2 cm).
 
-    Warnings
-    --------
-    The default `freq` is valid for the northern hemisphere.
-
     Parameters
     ----------
     snd : xarray.DataArray
         Surface snow thickness.
     thresh : Quantified
         Threshold snow thickness.
     freq : str
-        Resampling frequency.
+        Resampling frequency. The default value is chosen for the northern hemisphere.
     op : {">", ">=", "gt", "ge"}
         Comparison operation. Default: ">=".
 
     Returns
     -------
     xarray.DataArray, [time]
-        Number of days where snow depth is greater than or equal to threshold.
+        Number of days where snow depth is greater than or equal to {thresh}.
```

Comment:
Do the curly brackets work here? I'm guessing only for the Indicator object.

Reply:
The first line below the type and dimensionality is parsed as the "long_name" when the indicator doesn't declare its own. So they work!

See https://github.com/Ouranosinc/xclim/blob/main/docs/notebooks/Indice.svg